CONTRIBUTING.md

```
@@ -0,0 +1,36 @@
+# Contributing to mongoq
+
+This is a short guide on how to contribute to the project.
+
+## Submitting a pull request
+
+If you find a bug that you'd like to fix, or a new feature that you'd like to implement then please submit a pull request via GitHub.
+
+
+Fork the Repository:
+1. Visit https://github.com/kashifkhan0771/mongoq
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)